tests/unit/providers/test_anthropic_messages.py

```
@@ -618,8 +618,8 @@ async def handler(request: httpx.Request) -> httpx.Response:
 async def test_amessages_beta_extra_header_suppresses_unknown_edit_warning(
     caplog: pytest.LogCaptureFixture,
 ) -> None:
-    async def handler(request: httpx.Request) -> httpx.Response:
-        return httpx.Response(
+    async def handler(request: httpx2.Request) -> httpx2.Response:
```

Ruff (0.16.2)

[warning] Unused function argument: `request`

(ARG001)